thirdparty/open62541/open62541.patch

```diff
@@ -11,11 +11,20 @@ index d426e1da..5f1a4044 100644
  find_package(PythonInterp REQUIRED)
  find_package(Git)
  include(AssignSourceGroup)
-@@ -416,17 +416,17 @@ if(NOT UA_COMPILE_AS_CXX AND (CMAKE_COMPILER_IS_GNUCC OR "x${CMAKE_C_COMPILER_ID
+@@ -526,7 +526,7 @@ if(NOT UA_FORCE_CPP AND (CMAKE_COMPILER_IS_GNUCC OR "x${CMAKE_C_COMPILER_ID}" ST
+     check_add_cc_flag("-Wall")      # Warnings
+     check_add_cc_flag("-Wextra")    # More warnings
+     check_add_cc_flag("-Wpedantic") # Standard compliance
+-    check_add_cc_flag("-Werror")    # All warnings are errors
++    # check_add_cc_flag("-Werror")    # All warnings are errors
+ 
+     check_add_cc_flag("-Wno-static-in-inline") # Clang doesn't like the use of static inline methods inside static inline methods
```

Comment:
I think it's the use of static functions inside non-static inline functions that Clang doesn't like

Reply:
Actually I did not add this line, it was changed when updated to the latest version of OPC UA library. My change was only to remove the -Werror flag.